component/src/main/java/org/wso2/extension/siddhi/store/elasticsearch/ElasticsearchEventTable.java

```diff
@@ -845,43 +856,41 @@ private void createIndex() {
             XContentBuilder builder = XContentFactory.jsonBuilder();
             builder.startObject();
             {
-                builder.startObject(indexType);
```

Comment:
@dilini-muthumala , shall change the logic a bit and keep indexType? 
If user gives a value for indexType we'll have builder.startObject(indexType); if it's not we'll go for this PRs approach.

Reply:
+1. Added in 0522934